helm/bloop/values.yaml

```diff
@@ -185,11 +186,19 @@ qdrant:
 
   #modification example for configuration to overwrite defaults
   config:
+    telemetry_disabled: true
     cluster:
       enabled: true
       p2p:
         port: 6335
-      consensus:
+      concensus:
         tick_period_ms: 100
     storage:
       snapshots_path: /qdrant/storage/snapshots
+      on_disk_payload: false
```

Comment:
What's the effect of this on RAM? Not sure I understand this comment in Qdrant's config template: https://github.com/qdrant/qdrant/blob/7164ac4a5987d28f1c93f5712aef8e09e7d93555/config/config.yaml#L18

Reply:
Will probably increase RAM pressure significantly on large repos.